Source/core/Portability.h

```diff
@@ -684,7 +684,7 @@ typedef DWORD ThreadId;
 #define DEBUG_VARIABLE(x)
 #endif
 
```

Comment:
Let's add for the time being add a "namespace WPEFramework = Thunder;", we are of course done now but this will people some time to gradually transition